editor/src/components/editor/global-shortcuts.tsx

```diff
@@ -749,6 +759,24 @@ export function handleKeyDown(
             })
           : []
       },
+      [CONVERT_TO_FLEX_CONTAINER]: () => {
+        if (!isSelectMode(editor.mode)) {
+          return []
+        }
+        const selectedElementsFlexContainers = detectAreElementsFlexContainers(
+          editor.jsxMetadata,
+          editor.selectedViews,
+        )
+        const commands = commandsForFirstApplicableStrategy(
+          editor.jsxMetadata,
+          editor.selectedViews,
+          selectedElementsFlexContainers ? removeFlexLayoutStrategies : addFlexLayoutStrategies,
+        )
+        if (commands != null) {
+          return [EditorActions.applyCommandsAction(commands)]
```

Comment:
nit: just for consistency inside this function which keep the happy path to the left, I'd flip this condition and return `[]` here